### DISCUSSION TITLE
Unable to play with friends

### DISCUSSION THREAD

**Discussion by lLooshie** [DashieDev/DoggyTalentsNext#97] [Q&A]
I have come across a problem when my friend and I tried to play with this mode (version 1.21 - 1.18.27). For example, Training Treat only works when host player uses it, but when a guest obtains Training Treat it looks like Natto Rice or some other object and cannot use it to  convert an untamed Wolf into Trained Dog. Also every time host tries to level up their dog, guests get kicked out of the world. Hope these issues will be solved sometime!

**Comment by DashieDev** [comment]
Are you on neoforge or fabric ?

**Reply by lLooshie** [reply]
fabric
